SimFastTiming/FastTimingCommon/src/BTLTileDeviceSim.cc

```diff
@@ -0,0 +1,101 @@
+#include "SimFastTiming/FastTimingCommon/interface/BTLTileDeviceSim.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/ForwardDetId/interface/MTDDetId.h"
+#include "DataFormats/ForwardDetId/interface/BTLDetId.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "Geometry/CommonDetUnit/interface/GeomDetType.h"
+#include "Geometry/MTDGeometryBuilder/interface/ProxyMTDTopology.h"
+#include "Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h"
+
+#include "CLHEP/Random/RandGaussQ.h"
+
+BTLTileDeviceSim::BTLTileDeviceSim(const edm::ParameterSet& pset) : 
```

Comment:
This class could be mostly unified with `BTLBarDeviceSim` to reduce code duplication

Reply:
They correspond to two different detector concepts and future developments might differentiate them more. Or one of the two be simply dropped and removed. I would prefer to keep them distinct.